src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -288,34 +288,91 @@ public void UpdateStateForUI()
         StateChanged?.Invoke(this, GetState());
     }
 
+    private bool IsTerminalProvisioningState(string provisioningState)

```

Comment:
What does "Terminal" mean here and below? Might be good to add comments on both

Reply:
Added comment.